pom.xml

```diff
@@ -78,6 +81,25 @@
             <version>${liquibase.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>${hibernate-validator.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
```

Comment:
the same

To check if some dependency already provides other, you can call in console `mvn dependency:tree` and check if dependency already exists